examples/node/main.rs

```diff
@@ -169,6 +169,7 @@ fn main() {
 }
 
 fn init_logger() {
+    fs::create_dir_all(".ldk").unwrap();
```

Comment:
I would move this line above `init_logger();` since it is not only about logger initialization, but a directory for other things.